arangod/GeneralServer/H2CommTask.cpp

```diff
@@ -477,15 +477,13 @@ void H2CommTask<T>::processStream(H2CommTask<T>::Stream& stream) {
 
   // from here on we will send a response, the connection is not IDLE
   _numProcessing.fetch_add(1, std::memory_order_relaxed);
-
+  _url = std::string((req->databaseName().empty() ? "" : "/_db/" + req->databaseName())) +
```

Comment:
`_url` is set here and read only once a few lines later.
So we would not need an instance variable for it.
In the other CommTask implementations, `_url` is read twice and not just once. Are we missing the second place in the H2CommTask?

Reply:
true. fixed